pages/linux/last.md

```diff
@@ -0,0 +1,37 @@
+# last
+
+> List information of user logins.
+> See also: `lastb`, `tac`.
```

Comment:
Should `tac` be here? I understand that it's used in one of the examples, but its purpose is well-explained, and we are not using it in any fancy way.

It's kind of like `sudo`. We use it in examples, but including it in the description seems unnecessary.

Reply:
Now that I think about it, I agree. 

I thought it would be worth mentioning the `tac` command for other users.